_schemaV2.graphql

```diff
@@ -19284,6 +19275,27 @@ enum VerificationStatuses {
   VERIFIED_WITH_CHANGES
 }
 
+type VerifyAddressFailureType {
+  mutationError: GravityMutationError
+}
```

Comment:
This has me wondering - we made this a Query field early on after some discussion back and forth whether it was truly a query or a mutation, but now plan to back these queries up with a model in gravity. While we are changing the schema anyways, should we make this a mutation?

Reply:
I think it would be more straightforward as a mutation since we are performing an operation. A query implies that we are querying a type in our data model, which we aren't doing.

I also think that the `AddressVerificationFlow` component in Force will be greatly simplified if we were requesting a mutation and handling its response.

Having said that, I don't think there is a benefit to making that change today.

Reply:
Yeah I agree it makes more sense now to use a mutation for this operation, and agree we can do it in a followup, given it also requires some UI updates.